src/nvim/api/vimscript.c

```diff
@@ -282,7 +282,10 @@ Object nvim_call_dict_function(Object dict, String fn, Array args, Arena *arena,
   switch (dict.type) {
   case kObjectTypeString:
     TRY_WRAP(err, {
-      eval0(dict.data.string.data, &rettv, NULL, &EVALARG_EVALUATE);
+      int rv = eval0(dict.data.string.data, &rettv, NULL, &EVALARG_EVALUATE);
+      if (rv != OK && !ERROR_SET(err)) {
+        abort();
```

Comment:
Assume this never happens. If it does, then coverity was right and I need to restore the explicit handling removed in 022449b52236

Reply:
`:call nvim_call_dict_function('{', 'foo', [])`

Reply:
`err` has never been used up to this point (as `try_leave()` hasn't been executed yet), so `!ERROR_SET(err)` is always true here.